src/main/java/net/sf/jabref/logic/bibtexkeypattern/BibtexKeyPatternUtil.java

```diff
@@ -519,6 +521,13 @@ public static String applyModifiers(String label, List<String> parts, int offset
                     }
                     resultingLabel = abbreviateSB.toString();
 
+                } else if ("camel".equals(modifier)) {
+                    // CamelCasingOfEveryWord
+                    Title title = new Title(resultingLabel);
+                    title.getWords().stream().forEach(Word::toUpperFirst);
```

Comment:
I also reacted on that. There is quite a bit of more code than this relying on the current design though, so I add an issue and implement the `getWordsReadonly` thing.